app/services/permissions/user.rb

```diff
@@ -0,0 +1,30 @@
+module Permissions
+  module User
+    extend ActiveSupport::Concern
+
+    def can_edit?(org)
+      superadmin? || (!org.nil? && organisation == org)
+    end
+
+    def can_delete?(_org)
```

Comment:
@StlMaris123 I do not think you need this `_org` variable here, it's unused..

Reply:
you are correct